package.json

```diff
@@ -62,7 +62,7 @@
     "better-than-before": "1.0.0",
     "chai": "3.4.1",
     "chokidar-cli": "1.2.0",
-    "conventional-changelog-core": "1.0.1",
+    "conventional-changelog-core": "1.8.0",
```

Comment:
Great. Confirmed that this is just for testing. I guess this means that other packages upstream packages (like `conventional-changelog`) will need to use the latest version of `conventional-changelog-core` for *this* module to work correctly.